arduino/arduino-ide
# add axe to create accessibility reports

## Why
We want to make the Arduino IDE more accessible.

## How 
We decided to choose [axe](https://www.deque.com/axe/), a tool that does automated accessibility tests at runtime.

## Test instructions
- download the build and launch the executable
- open the developer tools (in the menu **Help > Toggle Developer Tools**)
- open the **Console** tab
- launch the command
```js
runAxe()
```
- you will see some coloured lines spawn in the console reporting accessibility issues, like this:
![image](https://user-images.githubusercontent.com/6939054/153278448-e08c563e-ce0f-49ca-b901-5094f0c1d43b.png)
- once you've run this command and until you close the window, as you interact with the application you will keep getting new reports
